iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.ComponentModel;
+using Microsoft.Azure.Devices.Registry;
```

Comment:
Why have sub namespaces? I'm pretty sure the Azure SDK team has advised us not to use so many when our API surface is not that large.

Reply:
Fair enough, we can keep it all in Microsoft.Azure.Devices